src/backend/parser/parse_relation.c

```diff
@@ -2517,17 +2483,15 @@ addRangeTableEntryForENR(ParseState *pstate,
 	{
 		case ENR_NAMED_TUPLESTORE:
 			rte->rtekind = RTE_NAMEDTUPLESTORE;
-			rte->requiredPerms = 0;
+			perminfo->requiredPerms = 0;
 			rte->inh = false;
 			break;
 
 		case ENR_TSQL_TEMP:
 			rte->inh = rv->inh;
 			rte->rtekind = RTE_RELATION;
 			rte->rellockmode = isLockedRefname(pstate, refname) ? RowShareLock : AccessShareLock;
-			rte->requiredPerms = ACL_SELECT;
-			rte->insertedCols = NULL;
-			rte->updatedCols = NULL;
+			perminfo->requiredPerms = ACL_SELECT;
```

Comment:
This is one section that needed to be manually modified in addition to the merge conflict